src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataResourceSerializer.cs

```diff
@@ -238,7 +241,7 @@ private async Task WriteDeltaResourceAsync(object graph, ODataWriter writer, ODa
             }
         }
 
-        private ResourceContext GetResourceContext(object graph, ODataSerializerContext writeContext)
+        internal ResourceContext GetResourceContext(object graph, ODataSerializerContext writeContext)
```

Comment:
Document internal method. In addition, the convention is to have internal methods above private methods